### DISCUSSION TITLE
Release v2.3.3

### DISCUSSION THREAD

**Discussion by androidacy-user** [Androidacy/MagiskModuleManager#74] [General]
**Changes**: [Read me](https://www.androidacy.com/androidacy-module-manager-download-v2-3-3-a-comprehensive-update-for-enhanced-module-management/?utm_source=github&utm_medium=web)

Downloads are linked in the release notes. 

<hr /><em>This discussion was created from the release <a href='https://github.com/Androidacy/MagiskModuleManager/releases/tag/v2.3.3'>Release v2.3.3</a>.</em>

**Comment by peggy-48** [comment]
BRO PUT IT IN GITHUB PAGES NO ADS ON WEBSITE PLSSS GIVE ORIGINAL REPO URL PLSSSSSS

**Reply by androidacy-user** [reply]
> BRO PUT IT IN GITHUB PAGES NO ADS ON WEBSITE PLSSS GIVE ORIGINAL REPO URL PLSSSSSS

No plans at this time, sorry

**Reply by Rikj000** [reply]
@androidacy-user, your site is a nightmare for privacy conscious users.

It's nearly impossible to get around the +-1500 3th party instances to which you sell our data,   
through hidden JavaScript "telemetry" (*cough* spy-ware *cough*)...

It's terrible to see @Fox2Code's great work being polluted by this.

If you don't seek to publish on Github / F-Droid,   
then I'll make sure to jump ship to the first active fork that sticks it's neck up :slightly_smiling_face: 

Me and my data, **will not be your product!**

**Reply by androidacy-user** [reply]
> @androidacy-user, your site is a nightmare for privacy conscious users.
> 
> It's nearly impossible to get around the +-1500 3th party instances to which you sell our data,   
> through hidden JavaScript "telemetry" (*cough* spy-ware *cough*)...
> 
> It's terrible to see @Fox2Code's great work being polluted by this.
> 
> If you don't seek to publish on Github / F-Droid,   
> then I'll make sure to jump ship to the first active fork that sticks it's neck up :slightly_smiling_face: 
> 
> Me and my data, **will not be your product!**

I apologize you feel that way, but you always have the option of removing ads for a coffee a month.

**Comment by Rikj000** [comment]
> I apologize you feel that way, but you always have the option of removing ads for a coffee a month.

Some FOSS development etiquette for you:
- User happy with the experience + Has funds to spare => Chance to receive a donation
- Not the other way around..
- Not by stealing/selling data from them if they don't have funds to spare,   
that's what evil proprietary big-tech companies do,   
which are slowly being replaced by fair FOSS alternatives (e.g. [Reddit](https://old.reddit.com/) => [Lemmy](https://join-lemmy.org/))

**Comment by androidacy-user** [comment]
> > I apologize you feel that way, but you always have the option of removing ads for a coffee a month.
> 
> Some FOSS development etiquette for you:
> - User happy with the experience + Has funds to spare => Chance to receive a donation
> - Not the other way around..
> - Not by stealing/selling data from them if they don't have funds to spare,   
> that's what evil proprietary big-tech companies do,   
> which are slowly being replaced by fair FOSS alternatives (e.g. [Reddit](https://old.reddit.com/) => [Lemmy](https://join-lemmy.org/))

Donations don't pay our bills. Donations don't compensate our moderation, our developers, and our support staff for their work. 

**People don't donate**. Period. It doesn't matter how great the experience is 99.99999% of users won't donate. People care far more and benefit far more from free as in beer rather than free as in freedom.

The reality is the day even 1% of our user base gives voluntarily, with no benefits to themselves, is the day we could remove ALL ads and ALL subscriptions. If even 1% donated $1 a month, we could have enough money to pay our bills and some left over.

We tried that model. And sure, for small single indie devs it might work. But it just doesn't fit us. If it works for you - great! If we like your work and benefit from it, we might even sponsor you. 

No disrespect intended, but we've thought this out and made some tough decisions along the way.

**Reply by Rikj000** [reply]
I did not realize that Fox's **non-profit ethical project**,   
fell into the hands of a **for-profit non-ethical organization** which sells user data to keep the lights on.

Donations don't pay my bills either,   
I write FOSS for humanity in my free time,   
and go to work during the day for my bills.

One can also argue with why your hosting expenses are so high.   
If you just linked original manager + module release downloads from Github, Gitlab, Gitea, Forgejo,   
instead of wiring us all through your spy-ware infested repo,   
then your monthly bandwidth would be much less.

But no, **we** all need to go through **your** repo, so **you** can earn money from:   
- The modules I and other developers have written
- The data you harvest of the users and then sell to whoever (1492 3th parties) offers you money for it

**Reply by androidacy-user** [reply]
> I did not realize that Fox's **non-profit ethical project**,   
> fell into the hands of a **for-profit non-ethical organization** which sells user data to keep the lights on.
> 
> Donations don't pay my bills either,   
> I write FOSS for humanity in my free time,   
> and go to work during the day for my bills.
> 
> One can also argue with why your hosting expenses are so high.   
> If you just linked original manager + module release downloads from Github, Gitlab, Gitea, Forgejo,   
> instead of wiring us all through your spy-ware infested repo,   
> then your monthly bandwidth would be much less.
> 
> But no, **we** all need to go through **your** repo, so **you** can earn money from:   
> - The modules I and other developers have written
> - The data you harvest of the users and then sell to whoever (1492 3th parties) offers you money for it

Fox gave up the project because of people like you stressing him out. People like you making this a culture war and trying to act like the ONLY ethical thing is giving everyone what they want all the time.

We make finding, browsing, downloading, and searching for modules easier, and pushing people off site would be counter to that

**Reply by Rikj000** [reply]
Please remove [my work](https://github.com/Rikj000/Magisk-Manager-for-Recovery-Mode) from your repo,   
I do not want my users to go through your spy-ware to get their download,   
I want them to get their downloads from the [official direct source](https://github.com/Rikj000/Magisk-Manager-for-Recovery-Mode/releases).

**Reply by androidacy-user** [reply]
You are trying to make this out as if you are some sort of hero for humanity and we are the enemy. We're not. We just chose a different path than you.

And fyi - the site you're using for those open source contributions? Makes profit from closed source. Linux kernel? Biggest contributions are from developers working for closed source, for profit companies. The world isn't as black and white as you seem to think. Open source is not the enemy of closed source, nor vice versa. We keep the original app open source out of respect and will keep the original app that way. 

We have staff, infrastructure, and various other costs to cover. We strive to balance generating revenue and user experience, and are always open to constructive feedback, but trying to paint us as evil ain't that.

**Reply by androidacy-user** [reply]
> Please remove [my work](https://github.com/Rikj000/Magisk-Manager-for-Recovery-Mode) from your repo,   
> I do not want my users to go through your spy-ware to get their download,   
> I want them to get their downloads from the [official direct source](https://github.com/Rikj000/Magisk-Manager-for-Recovery-Mode/releases).
> 
> 

Request granted. We regret you leaving, but we wish you luck in your future endeavors.
